cmd/gateway/s3x/gateway-s3x.go

```diff
@@ -180,8 +180,12 @@ func (g *TEMX) NewGatewayLayer(creds auth.Credentials) (minio.ObjectLayer, error
 	if err != nil {
 		return nil, err
 	}
-	go xobj.infoAPI.grpcServer.Serve(xobj.listener)
-	go xobj.infoAPI.httpServer.ListenAndServe()
+	go func() {
```

Comment:
No need to change this, it is equivalent to the the previous, but more verbose.

Reply:
It was giving me an lint error for missing err check.

Reply:
ah okay nevermind then